templates/pyramid/files/.platform/services.yaml

```diff
@@ -0,0 +1,6 @@
+mysql:
```

Comment:
I've been trying to avoid calling the mysql service `mysql`, as then you just end up with mysql repeated over and over. :smile: I think I've generally called it `mysqldb`.